examples/pruning/mxnet_integration.py

```diff
@@ -0,0 +1,124 @@
+"""
+Optuna example that demonstrates a pruner for MXNet.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+MXNet and MNIST, where the architecture of the neural network and the learning rate of optimizer
+is optimized. Throughout the training of neural networks, a pruner observes intermediate
+results and stops unpromising trials.
+
+You can run this example as follows:
+    $ python mxnet_integration.py
+
+"""
+
+import logging
+
+import mxnet as mx
+import numpy as np
+
+from optuna.integration import MXNetPruningCallback
+
+
+N_TRAIN_EXAMPLES = 3000
+N_TEST_EXAMPLES = 1000
+BATCHSIZE = 128
+EPOCH = 10
+
+# Set log level for MXNet.
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def create_model(trial):
+    # We optimize the number of layers, hidden units and dropout in each layer and
```

Comment:
This sentence seems incomplete. Besides, I think this function doesn't optimize dropout parameter.
```suggestion
    # We optimize the number of layers and hidden units in each layer.
```

Reply:
Thanks.

I referenced Keras example and tried to use Dropout...